### PR TITLE
MISUV-598: Covid Agents Kickout

### PR DESCRIPTION
**Pull request by AIbrahimQA**

### CODE REVIEW

File: conf/messages [hunk 1]
```diff
@@ -123,3 +123,19 @@ agent.income-sources-error.bullet3                                    = have a s
...
+agent.covid_cannot_sign_up.link                                       = Self Assessment tax return
+agent.covid_cannot_sign_up.button                                     = Sign up another client
+
+
+
+
+
+
+
+
```
Comment (AlexRimmerHMRC) [inline conf/messages:141]: can you remove the extra spacing

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/agents/covid_cannot_sign_up.scala.html [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    <h1>@messages("agent.covid_cannot_sign_up.heading")</h1>
+    <p>@messages("agent.covid_cannot_sign_up.line1")</p>
+    <p>
+    @Html(messages("agent.covid_cannot_sign_up.line2",
+            <a href="https://www.gov.uk/self-assessment-tax-returns/sending-return">{messages("agent.covid_cannot_sign_up.link")}</a>
+    ))
+    </p>
+    <form>
+        <button class="button" type="submit" formaction="@appConfig.signUpAnotherClientUrl" >@messages("agent.covid_cannot_sign_up.button")</button>
+    </form>
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/views/agents/covid_cannot_sign_up.scala.html:31]: Could you replace this form submit with how it is done in overview.scala.html formaction is only supposed to be for very specific use cases I think.

File: app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/config/AppConfig.scala [hunk 3]
```diff
@@ -50,6 +50,10 @@ class AppConfig @Inject()(config: ServicesConfig) {
 
   lazy val guidanceUrl: String = s"$govUkUrl/guidance/use-software-to-send-income-tax-updates"
 
+  lazy val returnsUrl: String = s"https://www.gov.uk/self-assessment-tax-returns/sending-return"
+
+  lazy val signUpAnotherClientUrl:  String = s"/report-quarterly/income-and-expenses/sign-up/eligibility/client/error/covid-cannot-sign-up"
```
Comment (AlexRimmerHMRC) [inline app/uk/gov/hmrc/incometaxsubscriptioneligibilityfrontend/config/AppConfig.scala:55]: We shouldn't need this line, we can get this url from reverse routes

File: conf/messages [hunk 4]
```diff
@@ -130,3 +130,11 @@ agent.income-sources-error.bullet3                                    = have a s
 agent.income-sources-error.para2.part1                                = If your client has other sources of income, they need to
 agent.income-sources-error.para2.link                                 = send a Self Assessment tax return
 agent.income-sources-error.para2.part2                                = instead.
+
+## Covid Cannot Sign Up page ##
+agent.covid_cannot_sign_up.title                                      = Your client cannot take part in this pilot
+agent.covid_cannot_sign_up.heading                                    = Your client cannot take part in this pilot
+agent.covid_cannot_sign_up.line1                                      = You will not be able to take part in this pilot on your client’s behalf if they have ever claimed a coronavirus (COVID-19 grant), or intend to do so in future.
```
Comment (AlexRimmerHMRC) [inline conf/messages:137]: Would you be able to check with design that this line is correct, seems weird to me that everywhere we mention coronavirus we have (COVID-19) but here specifically, grant is in the brackets. Even the welsh doesn't have it in the brackets.

Comment (AlexRimmerHMRC) [reply conf/messages:137]: I even checked the individual version of this exact line and grant isn't within the brackets, definitely  needs checking with design

Comment (AIbrahimQA) [reply conf/messages:137]: Yep asked gemma. Just waiting for a reply.

Comment (AIbrahimQA) [reply conf/messages:137]: yeah grants should not be in there. I will remove that now.

File: conf/messages [hunk 5]
```diff
@@ -130,3 +130,11 @@ agent.income-sources-error.bullet3                                    = have a s
 agent.income-sources-error.para2.part1                                = If your client has other sources of income, they need to
 agent.income-sources-error.para2.link                                 = send a Self Assessment tax return
 agent.income-sources-error.para2.part2                                = instead.
+
+## Covid Cannot Sign Up page ##
+agent.covid_cannot_sign_up.title                                      = Your client cannot take part in this pilot
+agent.covid_cannot_sign_up.heading                                    = Your client cannot take part in this pilot
+agent.covid_cannot_sign_up.line1                                      = You will not be able to take part in this pilot on your client’s behalf if they have ever claimed a coronavirus (COVID-19), or intend to do so in future.
```
Comment (AlexRimmerHMRC) [inline conf/messages:137]: I noticed you changed this, but it no longer makes sense. "if they have ever claimed a coronavirus" is there supposed to be the word grant after the (COVID-19) brackets?

Comment (AIbrahimQA) [reply conf/messages:137]: Gemma said that it should just be (COVID-19)- and that it must just be a mistake in the prototype as the Welsh document reflects these.

Comment (AIbrahimQA) [reply conf/messages:137]: but yeah it does seem weird. i will message her again.

Comment (AlexRimmerHMRC) [reply conf/messages:137]: I would be expecting it to be "if they have ever claimed a coronavirus (COVID-19) grant, or intended...

**Pull request merged**